link.sh

```diff
@@ -0,0 +1 @@
+npm link @daostack/client
```

Comment:
for what this file is for ?
should it be @daostack/arc.js ?

Reply:
Just testing :P